test/integration/smart_answer_flows/check_uk_visa_test.rb

```diff
@@ -7,11 +7,42 @@ class CheckUkVisaTest < ActiveSupport::TestCase
   include FlowTestHelper
 
   setup do
-    @location_slugs = %w(andorra anguilla armenia bolivia canada china colombia croatia mexico south-africa stateless-or-refugee syria turkey yemen oman united-arab-emirates qatar taiwan venezuela)
+    @location_slugs = %w(afghanistan andorra anguilla armenia bolivia canada china colombia croatia mexico south-africa stateless-or-refugee syria turkey yemen oman united-arab-emirates qatar taiwan venezuela)
```

Comment:
Minor: Unless it's critical, I'd be tempted to use one of the countries already in this list in your new tests, so you don't have to add `afghanistan` to the list.